src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java

```diff
@@ -0,0 +1,117 @@
+            }
+        }
+    }
+
+    private void sendGuildInfo(Guild guild, TextChannel channel){
+        EmbedBuilder eb = new EmbedBuilder();
+        eb.setTitle(guild.getName());
+        eb.setThumbnail(guild.getIconUrl());
+        eb.addField("Users", "**Total:** " +
+                guild.getMembers().size() + "\n" +
+                "\n" +
+                "**Online:** " +
+                guild.getMembers().stream().filter(member -> !member.getOnlineStatus().equals(OnlineStatus.OFFLINE)).count() + "\n" +
+                "\n" +
+                "**Owner:** " +
+                guild.getOwner().getUser().getName() +  "#" + guild.getOwner().getUser().getDiscriminator(), true);
+        String afk = guild.getAfkChannel() == null ? "" :
+                "**AFK:**\n" +
+                        "Channel: " +
+                        guild.getAfkChannel().getName() + "\n" +
+                        "\n" +
+                        "Timeout: " +
+                        FlareBot.getInstance().formatTime(guild.getAfkTimeout().getSeconds(), TimeUnit.SECONDS, true, false);
+        eb.addField("Channels", "**Text**\n" +
+                "Total: " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "Default: " +
+                "#" + guild.getTextChannelById(guild.getId()).getName() + "\n" +
+                "\n" +
+                "**Voice**\n" +
+                "Total: " +
+                guild.getVoiceChannels().size() + "\n" +
+                "\n" +
+                "Active: " +
+                guild.getVoiceChannels().stream().filter(voiceChannel -> voiceChannel.getMembers().size() > 0).count() + "\n" +
+                "\n" + afk, true);
+        eb.addField("Misc info", "**Creation time:** " +
+                guild.getCreationTime().toLocalDateTime().atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")) + " GMT/BST\n" +
+                "\n" +
+                "**Roles:** " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "**Server region:** " +
+                guild.getRegion().getName() + "\n" +
+                "\n" +
+                "**Verification Level:** " +
+                getVerificationString(guild.getVerificationLevel()), true);
+        eb.setFooter("ID: " + guild.getId(), null);
+        eb.setColor(Color.BLUE);
+        channel.sendMessage(eb.build()).queue();
+    }
+
+    private String getVerificationString(Guild.VerificationLevel level){
+        switch (level){
+            case HIGH:
+                return "(\u256F\u00B0\u25A1\u00B0\uFF09\u256F\uFE35 \u253B\u2501\u253B"; //(╯°□°）╯︵ ┻━┻
+            case VERY_HIGH:
+                return "\u253B\u2501\u253B\u5F61 \u30FD(\u0CA0\u76CA\u0CA0)\u30CE\u5F61\u253B\u2501\u253B"; //┻━┻彡 ヽ(ಠ益ಠ)ノ彡┻━┻
+            default:
+                return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
```

Comment:
The method I referenced does the exact same thing as you do here, the name is misleading I know